### DISCUSSION TITLE
ISO/IEC 27002:2021 keyword statistics

### DISCUSSION THREAD

**Discussion by K8Sudo** [monarc-project/MonarcAppFO#361] [Ideas]
With the new version of ISO/IEC 27002:2021, which is currently available in "DIS", some keywords are added to each control (control type, information security properties, cybersecurity concepts, operational capabilities, security domains). It would be helpful if monarc could perform reports on these keywords via the dashboard or the new Stats service.

Are there already possibilities in planning?

Greetings
Marc

**Comment by danje57** [comment]
I hope that the referential will be adapted consequently.
However I don't know how it will be feasible without recreating the analysis from the beginning...

**Comment by jerolomb** [comment]
Sorry, i have missed your comment, we have planned to update the referential. For the keyword we have to search an elegant manner to do that :) Because not all the referential have that

**Comment by jerolomb** [comment]
Hi here,

I hope it will help you https://www.youtube.com/watch?v=N45p3fw_G2U

It will not answer to the keyword issue but in the integration of the new version

Have a nice day 

Jérôme

**Reply by danje57** [reply]
Very nice and easy!!
